README.md

```diff
@@ -228,6 +228,21 @@ It simply chooses which version of the data to seed (the Snowflake copy should c
 **Args:**
 * `seed_name` (required): Name of the seed that has separate snowflake seed data.
 
+----
+### seed_data_helper ([source](macros/seed_data_helper.sql))
+This macro is intended to be used when a source table column is a reserved keyword in a warehouse, and Circle CI is throwing a fit.
+It simply chooses which version of the data to seed.
+
+***Usage:**
+```yml
+    # in integration_tests/dbt_project.yml
+    vars:
+        table_name: "{{ fivetran_utils.seed_data_helper(seed_name='user_data', warehouse='postgres') }}"
```

Comment:
would we be able to use this if there was a reserved keyword in both snowflake and postgres?

Reply:
not sure if having the warehouse as an argument works here

Reply:
This would not work if a reserved word was in both snowflake and postgres. That is a good point and something this macro should be able to handle. 

I have updated the macro to allow the `warehouses` argument to be a list and you may pass in both snowflake and postgres which the macro can then capture the snowflake and postgres specific seed data if needed!